Makefile

```diff
@@ -35,10 +35,12 @@ gobuild:
 static:
 	./scripts/build
 
+.dockerbuild:
```

Comment:
Why `.`?

Reply:
I had thought .target means 'target' is usually executed as a dependency of another target. I can change this.

Reply:
I haven't seen this convention, but @nmeyerhans might know more than me about this.